packages/@react-spectrum/autocomplete/src/SearchAutocomplete.tsx

```diff
@@ -40,6 +40,10 @@ import {useSearchAutocomplete} from '@react-aria/autocomplete';
 function SearchAutocomplete<T extends object>(props: SpectrumSearchAutocompleteProps<T>, ref: FocusableRef<HTMLElement>) {
   props = useProviderProps(props);
 
+  if (props.placeholder) {
+    console.warn('Placeholders are deprecated due to accessibility issues. Please use help text instead. See the docs for details: https://react-spectrum.adobe.com/react-spectrum/SearchAutocomplete.html#help-text');
```

Comment:
Perhaps drop the link here since the SearchAutocomplete docs aren't exposed on prod docs yet. Or maybe link to a built version of the docs?

Reply:
Good catch, I'll omit the link for now.